src/modules/home/components/property/property-card.tsx

```diff
@@ -57,6 +60,11 @@ export function PropertyCard({ id, name, location, pricePerNight, images, isAdde
     }
   };
 
+  const reviews = data?.data.reviews || [];
+  const totalRating = reviews.reduce((sum, review) => {
+    return sum + Number(review.rating);
+  }, 0);
```

Comment:
don't calculate this on frontend. on backend there is an example already. just use that and add additional overallRating field to response